packages/@aws-cdk/aws-events/lib/rule.ts

```diff
@@ -138,6 +140,58 @@ export class Rule extends Resource implements IRule {
 
     const roleArn = targetProps.role ? targetProps.role.roleArn : undefined;
 
+    if (targetProps.target) {
+      const targetStack = Stack.of(targetProps.target);
+      const sourceStack = Stack.of(this);
+      const sourceAccount = sourceStack.account;
+      if (targetStack.account !== sourceAccount) {
+        // cross-account event - strap in, this works differently than regular events!
+        // based on:
+        // https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/CloudWatchEvents-CrossAccountEventDelivery.html
+
+        // the _actual_ target is just the event bus of the target's account
+        this.targets.push({
+          id,
+          arn: `arn:aws:events:${targetStack.region}:${targetStack.account}:event-bus/default`,
```

Comment:
Use `targetStack.formatArn()`, this one is wrong in China (partition would be wrong).

Reply:
Yep. Good catch.

Reply:
Fixed.